src/cpu/cp0/reg_config.rs

```diff
@@ -7,6 +7,7 @@ pub struct RegConfig {
     endianness: Endianness,
 
     cu: bool,
+    kseg0_cache_enable_bits: [bool;3],
```

Comment:
TINY thing, but I would've added a space like this: `[bool; 3]`. Not important though; I'll accept this anyways :)